src/svelte/components/form/FormErrorSummary.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  export let errors: UIError[] = []
```

Comment:
```suggestion
  export let errors: ErrorDetail[] = []
```
There is an interface for the error in `types.d.ts` `ErrorDetail`